### ISSUE TITLE
Error install on Redmine 5.0.5-stable

### ISSUE THREAD

**Issue by jmeiracorbal** [alexbevi/redmine_knowledgebase#410]
Error on execute bundle install and migration:

```text
 [redmine 11/11] RUN bundle exec rake redmine:plugins:migrate RAILS_ENV=production:
3.307 rake aborted!
3.343 LoadError: cannot load such file -- macros
3.343 /usr/local/bundle/gems/zeitwerk-2.6.8/lib/zeitwerk/kernel.rb:38:in `require'
3.343 /usr/local/bundle/gems/zeitwerk-2.6.8/lib/zeitwerk/kernel.rb:38:in `require'
3.343 /usr/src/redmine/plugins/redmine_knowledgebase/init.rb:2:in `<top (required)>'
3.343 /usr/src/redmine/lib/redmine/plugin_loader.rb:31:in `load'
3.343 /usr/src/redmine/lib/redmine/plugin_loader.rb:31:in `run_initializer'
3.343 /usr/src/redmine/lib/redmine/plugin_loader.rb:108:in `each'
3.343 /usr/src/redmine/lib/redmine/plugin_loader.rb:108:in `block in load'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:427:in `instance_exec'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:427:in `block in make_lambda'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:198:in `block (2 levels) in halting'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:604:in `block (2 levels) in default_terminator'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:603:in `catch'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:603:in `block in default_terminator'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:199:in `block in halting'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:512:in `block in invoke_before'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:512:in `each'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:512:in `invoke_before'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/callbacks.rb:105:in `run_callbacks'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/reloader.rb:88:in `prepare!'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/application/finisher.rb:124:in `block in <module:Finisher>'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/initializable.rb:32:in `instance_exec'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/initializable.rb:32:in `run'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/initializable.rb:61:in `block in run_initializers'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/initializable.rb:60:in `run_initializers'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/application.rb:391:in `initialize!'
3.343 /usr/src/redmine/config/environment.rb:16:in `<top (required)>'
3.343 /usr/local/bundle/gems/zeitwerk-2.6.8/lib/zeitwerk/kernel.rb:38:in `require'
3.343 /usr/local/bundle/gems/zeitwerk-2.6.8/lib/zeitwerk/kernel.rb:38:in `require'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/dependencies.rb:332:in `block in require'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/dependencies.rb:299:in `load_dependency'
3.343 /usr/local/bundle/gems/activesupport-6.1.7.2/lib/active_support/dependencies.rb:332:in `require'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/application.rb:367:in `require_environment!'
3.343 /usr/local/bundle/gems/railties-6.1.7.2/lib/rails/application.rb:533:in `block in run_tasks_blocks'
3.343 /usr/local/bin/bundle:25:in `load'
3.343 /usr/local/bin/bundle:25:in `<main>'
3.344 Tasks: TOP => redmine:plugins:migrate => environment
3.344 (See full trace by running task with --trace)
3.348 W, [2023-07-26T09:44:19.349325 #8]  WARN -- : Creating scope :system. Overwriting existing method Enumeration.system.
```

**Comment by susumuf** [comment]
#401 helps you I guess.

**Comment by alexbevi** [comment]
This should be addressed by #411
